standard/types.md

```diff
@@ -546,7 +546,7 @@ A closed type is a type that is not an open type.
 
 At run-time, all of the code within a generic type declaration is executed in the context of a closed constructed type that was created by applying type arguments to the generic declaration. Each type parameter within the generic type is bound to a particular run-time type. The run-time processing of all statements and expressions always occurs with closed types, and open types occur only during compile-time processing.
 
-Each closed constructed type has its own set of static variables, which are not shared with any other closed constructed types. Since an open type does not exist at run-time, there are no static variables associated with an open type. Two closed constructed types are the same type if they are constructed from the same unbound generic type, and their corresponding type arguments are the same type.
+Each closed constructed type has its own set of static variables, which are not shared with any other closed constructed types. Since an open type does not exist at run-time, there are no static variables associated with an open type. Two closed constructed types are runtime interchangeable (§10.2.2) if they are constructed from the same unbound generic type, and an identity conversion exists between their corresponding type arguments.
```

Comment:
Is there the possibility of runtime interchangeable unbound generic types? I can't think of any offhand, but I thought I'd mention it.

> and an identity conversion exists between their corresponding type arguments

Do we need to be more precise here about each corresponding pair of type arguments? (We're not really talking about a single identity conversion, but about N conversions for N type arguments in each constructed type.)

Reply:
I re-organized this paragraph to address @KalleOlaviNiemitalo 's comment.

Reply:
> Do we need to be more precise here about each corresponding pair of type arguments? (We're not really talking about a single identity conversion, but about N conversions for N type arguments in each constructed type.)

Very likely. I struggled with formal language here. Effectively, what we need is the conversion analog of "it's turtles all the way down". For each corresponding type argument, there must be an identity conversion between the corresponding types. Each of the type arguments may be other closed generic types or tuples, which requires another cycle down.